site/includes/nav.pug

```diff
@@ -2,7 +2,7 @@
   nav
     ul.spectrum-SideNav.spectrum-SideNav--multiLevel
       li.spectrum-SideNav-item(class=pageURL === 'getting-started.html' ? 'is-selected' : '')
```

Comment:
I'm not seeing `is-selected` added to the docs page specifically 🤔 I thought this might have to do with the changes to resources/js/site.js but it doesn't look like that logic would've been affected by those changes

Reply:
I opted to pull those updates out as they feel unrelated to the focus of the PR